runners/dmn-tck-runner-drools/src/test/java/org/omg/dmn/tck/runner/drools/DroolsTCKTest.java

```diff
@@ -57,9 +71,49 @@
    implements DmnTckVendorTestSuite
 {
 
+   private static final SimpleFileVisitor<Path> rmrf = new SimpleFileVisitor<Path>() {
+      @Override
+      public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IOException {
+         Files.delete(file);
+         return FileVisitResult.CONTINUE;
+      }
+
+      @Override
+      public FileVisitResult postVisitDirectory(Path dir, IOException exc) throws IOException {
+         Files.delete(dir);
+         return FileVisitResult.CONTINUE;
+      }
+   };
    private static final Logger logger = LoggerFactory.getLogger(DroolsTCKTest.class);
    public static final BigDecimal NUMBER_COMPARISON_PRECISION = new BigDecimal("0.00000001");
 
+   File propsFile;
+   Properties props;
+
+   {
+      File dir = new File("../../TestResults/Drools");
+      String[] versions = dir.list((f, s) -> f.isDirectory());
+      assert versions != null;
+      Arrays.sort(versions);
```

Comment:
This is a bit problematic for instance it is meant to handle the case of multiple versions.
Shall we end up with `7.7.0.Final`, ... `7.9.0.Final`  and `7.10.0.Final` It would erase 7.10 and keep 7.9